### DISCUSSION TITLE
Really bad bed level?

### DISCUSSION THREAD

**Discussion by Avpman2** [OrcaSlicer/OrcaSlicer#5294] [Q&A]
What is wrong with this bed? Zaxis, leveling, and? Best way to correct? Sorry, I'm clueless at the moment, but l'm becoming more experienced every day.

![zaxis](https://github.com/SoftFever/OrcaSlicer/assets/3275462/9dda85fa-fd16-4aea-825d-132fc5db4035)

**Comment by darco415** [comment]
Ya asking on a slicer forum isnt the way since it has nothing to do w/your hw. Trying going on Reddit , maybe your printer mfg  has a sub there. It will prompt you to answer a few questions..good luck
